public/add.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./add.css" class="css" type="text/css">
+  <title>Page 2</title>
+</head>
+<body>
+  <header>
+    <h2>ADD PAGE</h2>
+  </header>
+  <input type="text" placeholder="Input your goal here...">
+  <div>
+    <label for="date">Slide to set your deadline or input it here</label>
```

Comment:
I think it would be good to put both the label, the date input and the date slider inside one container so that its clear to the user that that entire section is focused on the deadline.